import-export-cli/integration/testutils/policy_testUtils.go

```diff
@@ -0,0 +1,389 @@
+	customPolicySubType       = "custom rule"
+	subscriptionPolicySubType = "subscription policy"
+)
+
+// ValidateThrottlePolicyExportImport : Validates Exporting Throttling Policy from source env and Importing to destination env
+func ValidateThrottlePolicyExportImport(t *testing.T, args *PolicyImportExportTestArgs, adminUsername, adminPassword, policyType string) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+	base.SetupEnv(t, args.DestAPIM.GetEnvName(), args.DestAPIM.GetApimURL(), args.DestAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	policyName := fmt.Sprintf("%v", args.Policy[policyNameKey])
+
+	exportedOutput, err := exportThrottlePolicy(t, policyName, args)
+	assert.Nil(t, err, "Error while exporting the Throttling Policy")
+
+	args.ImportFilePath = base.GetExportedPathFromOutput(exportedOutput)
+	assert.True(t, base.IsFileAvailable(t, args.ImportFilePath))
+	args.SrcAPIM.DeleteThrottlePolicy(fmt.Sprintf("%v", args.Policy[policyIDKey]), policyType)
+
+	// Import Throttling Policy to env 2
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	importedOutput, err := importThrottlePolicy(t, args)
+	assert.Nil(t, err, "Error while importing the Throttling Policy")
+	assert.Contains(t, importedOutput, "Successfully imported")
+	// Give time for newly imported Throttling Policy to get indexed
+	base.WaitForIndexing()
+
+	// Get Throttle Policy from env 2
+	importedPolicy, _ := getThrottlingPolicyByName(t, args, adminUsername, adminPassword, policyName, policyType)
+	// Validate env 1 and env 2 policy is equal
+	validatePoliciesEquality(t, true, args, importedPolicy)
+}
+
+// ValidateThrottlePolicyImportUpdate : Validates importing existing Throttling Policy
+func ValidateThrottlePolicyImportUpdate(t *testing.T, args *PolicyImportExportTestArgs, adminUsername, adminPassword, policyType string) {
+	t.Helper()
+
+	// Setup apictl envs
+	base.SetupEnv(t, args.SrcAPIM.GetEnvName(), args.SrcAPIM.GetApimURL(), args.SrcAPIM.GetTokenURL())
+	base.SetupEnv(t, args.DestAPIM.GetEnvName(), args.DestAPIM.GetApimURL(), args.DestAPIM.GetTokenURL())
+
+	// Export policy from env 1
+	base.Login(t, args.SrcAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	policyName := fmt.Sprintf("%v", args.Policy[policyNameKey])
+
+	exportedOutput, _ := exportThrottlePolicy(t, policyName, args)
+	args.ImportFilePath = base.GetExportedPathFromOutput(exportedOutput)
+	assert.True(t, base.IsFileAvailable(t, args.ImportFilePath))
+	args.SrcAPIM.DeleteThrottlePolicy(fmt.Sprintf("%v", args.Policy[policyIDKey]), policyType)
+
+	// Import Throttling Policy to env 2
+	base.Login(t, args.DestAPIM.GetEnvName(), args.CtlUser.Username, args.CtlUser.Password)
+	importedOutput, err := importThrottlePolicy(t, args)
```

Comment:
Here inside args the Update flag is true, which is wrong. You should send it as false.

Reply:
fixed with 3c4e0a6955b4c9779a67ea315b574412c7967a6d